### ISSUE TITLE
design question about .hed-label

### ISSUE THREAD

**Issue by ryanpitts** [OpenNews/opennews-source#117]
@beep @kissane hoping for some feedback here ...

Was doing a test run on the "user logs in with email link and posts a new job" workflow, and it was noted that `.hed-label` over the login link felt like a button, and that was momentarily confusing, like you should be able to click there to perform the task. (I did change the language from "Post a job on Source" to just "Jobs on Source" to mitigate at least somewhat, with less call-to-actiony language.)

![screen shot 2017-02-08 at 5 03 49 pm](https://cloud.githubusercontent.com/assets/56477/22764769/a1b03a4e-ee21-11e6-8454-f6c15ac692cc.png)

The user also mentioned that they'd run into that same confusion on the homepage, with the "The Latest" label over the lead story, wanting to click that to get to an article list.

![screen shot 2017-02-08 at 5 12 14 pm](https://cloud.githubusercontent.com/assets/56477/22764797/d24f41e0-ee21-11e6-9072-e94faf8f29cb.png)

Wanted to raise this, even as late in the process as it is, in case there's anything we think might mitigate further. Things that come to mind, from least to greatest impact:

* stick with the current style
* double-check language in all labels, adjusting if necessary to make them feel less clickable
* change labels to links in more places than we currently do (starting with "The Latest," etc.) so that clickable is the default, and the ones that _aren't_ clickable look like the different ones
* adjust the `.hed-label` style to make them feel less like click targets

And maybe I'm missing some other options we might have. I'm not sure this is a blocker, but I thought it was worth raising in case any thoughts come to mind.

**Comment by kissane** [comment]
This is a legit concern. I feel like for launch, we're looking at double-checking the language and carrying on. And then maybe after launch we take a moment to link up more of them, and then probably look at a small adjustment to the style of the unlinked ones as well.

That is my thought.

**Comment by beep** [comment]
Yeah, I definitely could see how those little tags are getting overloaded. I’d originally included them [on the article page](http://sourcedev.ethanmarcotte.com/tmpl/article.php) as a bit of visual interest, but I think it makes sense to refine/rework them.

> I feel like for launch, we're looking at double-checking the language and carrying on. And then maybe after launch we take a moment to link up more of them, and then probably look at a small adjustment to the style of the unlinked ones as well.

I think this is a great plan. If you’d like, one quick, pre-launch adjustment might be to desaturate them a bit more. For example,

<img width="1440" alt="profile" src="https://cloud.githubusercontent.com/assets/124493/22782222/10755a78-ee94-11e6-94e1-169a133dc19e.png">

<img width="1280" alt="home" src="https://cloud.githubusercontent.com/assets/124493/22782225/1358d54e-ee94-11e6-8ba7-adb2e5307877.png">

Would that help, as a short-term fix? Happy to PR that, if’n you like it. (Or never mention it again, if’n you don’t!)

**Comment by ryanpitts** [comment]
It seems like a fine fix to me, although I'm not the right person at all to make color judgments. I trust! Also seems fine to consider this in a few hours.

@beep one thing I'm noticing is that I get a brief flash of the search box on pages when I look at https://opennews-source-staging.herokuapp.com/ on my phone and tablet. Is that just me?

**Comment by kissane** [comment]
I would like to officially consider this in a few hours, with much gratitude.

**Comment by beep** [comment]
> @beep one thing I'm noticing is that I get a brief flash of the search box on pages when I look at https://opennews-source-staging.herokuapp.com/ on my phone and tablet. Is that just me?

Hm, no—I’m seeing it too, @ryanpitts. I’ll open up a new issue.

> I would like to officially consider this in a few hours, with much gratitude.

👍👍👍

**Comment by ryanpitts** [comment]
the fixes referenced here are on prod, leaving the thread open to revisit more generally at a time that's not now
